SingularityService/src/main/java/com/hubspot/singularity/data/UsageManager.java

```diff
@@ -95,6 +104,10 @@ private String getCurrentSlaveUsagePath(String slaveId) {
     return ZKPaths.makePath(getSlaveUsagePath(slaveId), CURRENT_USAGE_NODE_KEY);
   }
 
+  private String getSpecificClusterUtilizationPath(long timestamp) {
```

Comment:
the "specific" keyword was the pattern the other items were using so I kept that since the timestamp specified which one to grab. I can drop the historical data and then rename it